pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -766,6 +766,16 @@ self: super: {
     '';
   });
 
+  # use master with patch that has build fixes
+  esqueleto = appendPatch (overrideCabal super.esqueleto (drv: {
+    src = pkgs.fetchFromGitHub {
```

Comment:
This patch pins our version of `esqueleto` unconditionally to an more or less arbitrary git commit plus patches that upstream has not approved in any way. This kind of override is a maintenance nightmare because it will silently break updates in the main `hackage-packages.nix` file.